winmerge.sls

```diff
@@ -1,12 +1,15 @@
+{% set version = '2.16.0' %}
+
 {% if grains['cpuarch'] == 'AMD64' %}
     {% set PROGRAM_FILES = "%ProgramFiles(x86)%" %}
 {% else %}
     {% set PROGRAM_FILES = "%ProgramFiles%" %}
 {% endif %}
+
 winmerge:
```

Comment:
Add a for loop here after the package name (`winmerge`):
```
winmerge:
{% for version in versions %}
```

Reply:
Put an endfor at the bottom of the file to close the loop:
```
    ...
    reboot: False
{% endfor %}
```
See this for an example:
https://github.com/saltstack/salt-winrepo-ng/blob/master/pidgin.sls